aerolyzer/image_restriction_functions.py

```diff
@@ -0,0 +1,215 @@
+'''
+Image Restriction Function File
+Description: This file contains all functions for the verifying image restrictions.
+'''
+
+import exifread
+import re
+import yaml
+
+class imgRestFuncs(object):
+    'Class containing all image restriction functions'
+
+    def __init__(self):
+        self.criteria = self._import_yaml("image_restriction_conf.yaml")
+
+    '''
+    Purpose:        The purpose of this function is to determine whether or not the device the
+                    image was taken on is an accepted mobile device.
+    Inputs:         dict exifData, list acceptedMobileDevices
+    Outputs:        None
+    Returns:        Boolean
+    Assumptions:    N/A
+    '''
+    def is_accepted_device(self, exifData):
+        if ('DEVICE' not in exifData.keys()):
+            return False
+        
+        if any(exifData['DEVICE'].lower() in self.device for self.device in self.criteria['acceptedMobileDevices']):
```

Comment:
?? Can you please double check these function definitions? I don't see a variable called self.device nor a 'DEVICE' tag when you use the exifread lib on some test image. Actually, please check all the tags in this document.